documentation-site/examples/datepicker/datepickers-color-states.js

```diff
@@ -3,6 +3,7 @@ import React from 'react';
 
 import {useStyletron} from 'baseui';
 import {FormControl} from 'baseui/form-control';
+import {Block} from 'baseui/block';
```

Comment:
do you mind replacing `Block` with `useStyletron`? I went through and removed all block calls from examples last week

Reply:
thanks @chasestarr - just fixed it